ui/static/ui/css/mit-lore.css

```diff
@@ -546,6 +561,10 @@ ul.repos {
     transform: translateY(-27%);
 }
 
+.nav-tabs {
+    border-bottom:1px solid #ddd !Important;
```

Comment:
This doesn't appear to require !important, at least in Firefox. Is it conflicting with something else?